addon/components/flexberry-tab-bar.js

```diff
@@ -0,0 +1,140 @@
+import Ember from 'ember';
+import layout from '../templates/components/flexberry-tab-bar';
+
+/**
+  Component's CSS-classes names.
+  JSON-object containing string constants with CSS-classes names related to component's .hbs markup elements.
+
+  @property {Object} flexberryClassNames
+  @property {String} flexberryClassNames.prefix Component's CSS-class names prefix ('flexberry-tab-bar').
+  @property {String} flexberryClassNames.wrapper Component's wrapping <div> CSS-class name ('flexberry-tab-bar').
+  @property {String} flexberryClassNames.tab Component's inner <input type="checkbox"> CSS-class name ('flexberry-tab-bar-tab').
+  @property {String} flexberryClassNames.tabIcon Component's inner <label> CSS-class name ('flexberry-tab-bar-tab-icon').
+  @readonly
+  @static
+
+  @for FlexberryTabBarComponent
+*/
+const flexberryClassNamesPrefix = 'flexberry-tab-bar';
+const flexberryClassNames = {
+  prefix: flexberryClassNamesPrefix,
+  wrapper: flexberryClassNamesPrefix,
+  tab: flexberryClassNamesPrefix + '-tab',
+  tabIcon: flexberryClassNamesPrefix + '-tab-icon'
+};
+
+/**
+ * FlexberryTabBarComponent
+ * Component to display semantic ui tabs
+ * @extends Ember.Component
+ */
+export default Ember.Component.extend({
+  classNames: ['flexberry-tab-bar'],
+
+  layout,
+
+  /**
+    Reference to component's CSS-classes names.
+    Must be also a component's instance property to be available from component's .hbs template.
+  */
+  flexberryClassNames,
+
+  /**
+   * Contains items to display in tab bar
+   * @property items
+   * @type {Array}
+   * @default []
+   * @example items: [{selector: 'tab1', caption: 'Tab one', active: true }, {selector: 'tab2', caption: 'Tab two'}]
+   * @desc the first item with active=true will be set as active, others ignored
+   */
+  items: [],
```

Comment:
I understand that in most cases another items array will be passed into component, but it is better to use null as default value for  such properties, and initialize them inside 'init ' hook like that `this.set('items', Ember.A())`. Besides pure JS arrays (`this.set('items', [])`) are not observable in ember applications (component won't react on changes in such array), so ember-array must be used instead (`this.set('items', Ember.A())`).